dotcom-rendering/src/components/Nav/ExpandedMenu/Columns.tsx

```diff
@@ -273,9 +275,14 @@ export const Columns = ({
 						data-link-name={nestedOphanComponents('nav2', 'search')}
 						className="selectableMenuItem"
 						tabIndex={-1}
+						id={searchId}
```

Comment:
If this is the only place this is used and we're not in a map this could just be a hardcoded id?

Reply:
Excellent idea 💡

Reply:
gu-search-[some-random-number]?

Reply:
`gu-search-mobile`